### PR TITLE
Clean up projectile

### PR DESCRIPTION
**Pull request by owenl131**
Clean up

### CODE REVIEW

File: Scripts/Projectile.cs [hunk 1]
```diff
@@ -33,51 +34,45 @@ public Projectile(float range, float speed, Vector3 direction, PackedScene proje
...
+        projectileLength = projectileInstance.GetNode<CSGBox>("CSGBox").Depth;
         projectileInstance.Connect("body_entered", this, "OnBodyEnteredProjectile");
-        // TODO change to lifetime
-        projectileInstance.GravityScale = 2;
+        projectileInstance.GravityScale = 0;
         // To make the bullet's origin at the edge of the projectile weapon
         Vector3 weaponFrontDirection = -GlobalTransform.basis.z;
-        Translation += projectileLength / 2 * weaponFrontDirection;
-        hit = false;
+        Translation += origin;
```
Comment (daniellimws) [inline Scripts/Projectile.cs]: What's `origin` initialized with?

**Review by daniellimws — CHANGES_REQUESTED** [review]

File: Scripts/Projectile.cs (hunk 1 above)
Comment (owenl131) [reply Scripts/Projectile.cs]: gg

Comment (daniellimws) [reply Scripts/Projectile.cs]: gg

Comment (owenl131) [reply Scripts/Projectile.cs]: may I have the origin pleaze

**Comment by daniellimws** [comment]
dotnet format failed

**Review by daniellimws — APPROVED** [review]
Ok can merge after fixing dotnet format

**Pull request merged**